### PR TITLE
Updates PCCS URL for edgelessdb + ego

### PR DESCRIPTION
**Pull request by otherview**
### Why is this change needed?

- https://github.com/obscuronet/obscuro-internal/issues/1273
- In Alibabaclound the PCCS must be specified as the default values don't work.
- Fixed docker-compose env var ingestion ( it was failing to load variables that were not immediately used in the entry command of the container)
- Enclave now starts with an` entry.sh` so it can handle the PCCS address ( needed for ali baba and if you have a custom remote attestation url)
- Enclave is no longer running under docker privileged mode ( changed to used devices like sugested by the edgeless systems )
- Remove the entry point in the docker files as it's misleading that the image may run without any flags

### What changes were made as part of this PR:

- Provide a high level list of the changes made

### Definition of done

- [ ] Unit tests added to cover new or changed functionality 
- [ ] Docs pages updated to cover new or changed functionality
- [ ] [Changelog.md](https://github.com/obscuronet/go-obscuro/blob/main/docs/testnet/changelog.md) updated 
- [ ] End-to-end tests run if required (see below)

### End-to-end tests

Running the end-to-end tests in the [obscuro-test repo](https://github.com/obscuronet/obscuro-test) is currently at the 
discretion of the developer prior to merging a PR. The intention is not to slow down development by having the longer 
running end-to-end tests as a PR gate run on each branch commit etc. To manually trigger a run pre-merge;

- Go to [run_local_tests](https://github.com/obscuronet/obscuro-test/actions/workflows/run_local_tests.yml)
- Click the "Run workflow" button
- Use the main branch of obscuro-test 
- Enter the name of the PR branch for go-obscuro
- Run the workflow

The run takes approximately 20 minutes and any failure will be reported in the workflow output. Should the tests fail 
the docker container output for the local testnet and all test artifacts will be added to the run and can be downloaded
for debugging and analysis, with a retention period of 2 days. 

Note that every PR merge to main will also trigger a run of post-merge tests, so even if you do not manually trigger 
pre-merge, tests will be run post-merge. The intention being to catch any issues fast on main to allow for quick 
resolution. The post-merge test output can be seen at 
[run_merge_tests](https://github.com/obscuronet/obscuro-test/actions/workflows/run_merge_tests.yml) and will show both 
the PR number and author in the list of runs.

### CODE REVIEW

File: go/enclave/main/entry.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+set -e
```
Comment (StefanIliev545) [inline go/enclave/main/entry.sh:2]: This one is confusing me, let's add a couple of comments here on what is happening and why for future references.

**Review by StefanIliev545 — CHANGES_REQUESTED** [review]
Looks great, just needs a couple of small comments.

File: go/enclave/main/entry.sh (hunk 1 above)
Comment (otherview) [reply go/enclave/main/entry.sh:2]: Good point ! Adding them 👍

**Review by StefanIliev545 — APPROVED** [review]
LGTM

**Pull request merged**